src/core/ipe-setup.c

```diff
@@ -0,0 +1,93 @@
+/* SPDX-License-Identifier: LGPL-2.1-or-later */
+
+#include "copy.h"
+#include "fd-util.h"
+#include "fileio.h"
+#include "ipe-setup.h"
+
+#define IPE_SECFS_DIR "/sys/kernel/security/ipe"
+#define IPE_SECFS_NEW_POLICY IPE_SECFS_DIR "/new_policy"
+#define IPE_SECFS_POLICIES IPE_SECFS_DIR "/policies/"
+#define IPE_POLICY_PATH "/etc/ipe/ipe-policy.p7b"
```

Comment:
? There seems to be a typo here

Reply:
Where? I can't see it, maybe I'm blind

Reply:
I've never heard of /etc. I'm pretty sure it's a typo and should be /usr

Reply:
Eheh this matches IMA - the policy is a local node thing, not an OS vendor thing, the OS vendor can build it in the kernel if they want a default one

Reply:
It seems clear that `/usr` is a typo for `/user` ;)

(Jokes aside, I think it's just the `p7b` suffix being weird here)

Reply:
p7b is the right suffix as it's binary+inline and it's what the documentation says to use
https://microsoft.github.io/ipe/usage/deployment/
https://en.wikipedia.org/wiki/PKCS_7#File_types